tests/unit/Change.spec.ts

```diff
@@ -48,7 +52,7 @@ describe('Change component', () => {
     'JiIUlDmKZ2ow7GmmDabic8igHnEDYD6sI7OFYnCJhRdgVEHN-_4KUk2YsAVl5XUr6blJKMuYDPeMyNreGTXU7foE4AT-93FwlyTyFzQGddrDv' +
     'c6kkQr7mgJNTtgg87DdYbVGbEtIetyVfvwEF0rU8JH2N-j36XIebo33FU3-gJ5Y5S69EHPqQ37R9H4d8WUrHO-4QzJQih3Yaea820XBplJeo0' +
     'DO3hQoVtPD42j0p3aIy10cnW2g')
```

Comment:
Check is KEYCLOAK_TOKEN needs to be declared here. I'm thinking, since you're mounting the Change component directly, and it doesn't do anything with the KC token, auth, etc, that this test suite can be simplified.

See App.vue::fetchData() -- this is the code that is NOT run when you mount your page instead of the entire app.

Reply:
I think it can be for the tests that I currently have running here. However I think these tests marked for future need them. https://github.com/bcgov/business-edit-ui/blob/59ab050c2e617017807bc8efa055ca8406e9f2f0/tests/unit/Change.spec.ts#L250

Reply:
I think the KC token is only used for the user's roles.

The other tests in this file do not mount App, so fetchData() will not be executed.